Zend/zend_exceptions.stub.php

```diff
@@ -56,8 +53,7 @@ function __wakeup() {}
     /** @return string */
     final function getTraceAsString() {}
 
-    /** @return string */
-    function __toString() {}
+    function __toString(): string {}
```

Comment:
Isn't this a BC break that has not been specified in the RFC? https://3v4l.org/9jlIJ

Reply:
No, it isn't. __toString() has been changed to be implicitly string in the compiler - so it will be implicitly compatible.